### PR TITLE
Messages migration files

### LINKED ISSUE #43: Messages - Migration files
Pre-injecting test data to the Message app data model.

### PR DESCRIPTION
**Pull request by Gal-f**
Adding migration files with mock data to the messages app. 
Fix #43

### CODE REVIEW

**Comment by ravidbro** [comment]
Looks OK overall but you need to fix the tests

File: forumMessages/migrations/0002_initial.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Generated by Django 3.1.4 on 2020-12-11 12:38
+
+from django.db import migrations, models, transaction
+from django.utils import timezone
+from django.contrib.auth.models import User
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('forumMessages', '0001_initial.py'),
```
Comment (simzacks) [inline forumMessages/migrations/0002_initial.py]: This should be '0001_initial' without the .py
See https://docs.djangoproject.com/en/3.1/topics/migrations/#migration-files

**Comment by lmilbaum** [comment]
@Gal-f Are you still working on this PR?

**Comment by ravidbro** [comment]
@Gal-f is this PR still relevant?

**Comment by Gal-f** [comment]
> @Gal-f is this PR still relevant?

Yes. It only requires a minor syntax fix,
I've made this a draft in order to not attract attention to it while I was unavailable to work on this.

**Review by ravidbro — APPROVED** [review]

**Pull request merged**